packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -32,7 +32,7 @@ const SOKOL = {
   rpcNode: 'https://sokol.poa.network',
   rpcArchiveNode: 'https://sokol-archive.blockscout.com',
   rpcWssNode: 'wss://sokol.poa.network/wss',
-  relayServiceURL: 'https://relay-staging.stack.cards/api',
+  relayServiceURL: 'http://localhost:8000/api',
```

Comment:
```suggestion
  relayServiceURL: 'https://relay-staging.stack.cards/api',
```

Should this be reverted back?